src/main.go

```diff
@@ -137,3 +149,20 @@ func getIdToType(podData *vehicle_models.PodData) map[uint16]string {
 	}
 	return idToType
 }
+
+func getConfig() Config {
+	configFile, fileErr := os.ReadFile("./config.toml")
```

Comment:
instead of hardcoding the config file, add a flag to get it.

A better way would be to have this method get as a parameter the config file path